.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "consistent-return": "off",
```

Comment:
- It doesn't allow `async` functions without `return`... that's like half of the _magic_ of `async` functions.